Sources/CryptomatorCloudAccess/Dropbox/DropboxCloudProvider.swift

```diff
@@ -24,6 +24,8 @@ public class DropboxCloudProvider: CloudProvider {
 		switch dropboxError {
 		case .tooManyWriteOperations, .internalServerError, .rateLimitError:
 			return true
+		case .authError:
+			return false
```

Comment:
I don't believe that this change does anything. Should already work, since the default case already returns false. I'd suggest reverting it.